src/query/api/v1/middleware/rewrite.go

```diff
@@ -42,6 +43,8 @@ import (
 	xtime "github.com/m3db/m3/src/x/time"
 )
 
+var errIgnorableQuerierError = fmt.Errorf("ignorable error")
```

Comment:
Nit: `fmt.Errorf` is meant for formatting, a straight `errors.New` would be slightly more preferred here.